CoreFoundation/String.subproj/CFString.c

```diff
@@ -5139,12 +5139,13 @@ void CFStringLowercase(CFMutableStringRef string, CFLocaleRef locale) {
 
     if (!langCode && isEightBit) {
         uint8_t *contents = (uint8_t *)__CFStrContents(string) + __CFStrSkipAnyLengthByte(string);
-        for (;currentIndex < length;currentIndex++) {
+        while (currentIndex < length) {
             if (contents[currentIndex] >= 'A' && contents[currentIndex] <= 'Z') {
                 contents[currentIndex] += 'a' - 'A';
             } else if (contents[currentIndex] > 127) {
                 break;
             }
+	currentIndex++;
```

Comment:
Looks to not be indented to the right level.

While I understand that a `for` loop without the initialization condition looks weird, I still think it shows the intent better than a `while` loop and increasing the index inside the loop.

Reply:
This also has not been resolved. Please revert this loop to the original version.